### PR TITLE
Pinned property value in Json Schema

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

File: pages/workspace/tracking-plan/publishing.mdx [hunk 1]
```diff
@@ -448,13 +448,14 @@ Json representation of your tracking plan, following the [Json Schema](https://j
...
-| type         | string             | Property type                                 |
-| nameMappings | array(NameMapping) | Name mappings defined for given property      |
-| ?enum        | array(string)      | Allowed enum values                           |
-| ?operations  | array(string)      | Array of property ops, e.g. "SnowplowContext" |
+| Attribute    | Type                         | Description                                   |
+| :----------- | :--------------------------- | :-------------------------------------------- |
+| description  | string                       | Property description                          |
+| type         | string                       | Property type                                 |
+| nameMappings | array(NameMapping)           | Name mappings defined for given property      |
+| ?const       | Depends on the property type | Pinned property value                         |
```
Comment (logason) [inline pages/workspace/tracking-plan/publishing.mdx:456]: This line is the only change, rest is formatting fix

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…acking-plan/publishing**](https://docs-gyao0d3m0.avo-dev.app/workspace/tracking-plan/publishing) |
|:-:|
|<a href="https://docs-gyao0d3m0.avo-dev.app/workspace/tracking-plan/publishing">Screenshot of /workspace/tracking-plan/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/XwSUCKhJqRbawPQqh1BfDqgE5kWWI95sYtQXPPgux2XREFRMUNU5_ihh2s2O7X0ktEyz4r26pRrYz6cGZVtbAw.png">(view full size)</a> |







Commit f066ce96daf513b129f6bcf2f8d5ad5dbf188ebf (https://docs-gyao0d3m0.avo-dev.app).

File: pages/workspace/tracking-plan/publishing.mdx (hunk 1 above)
Comment (alex-tpom6oh) [reply pages/workspace/tracking-plan/publishing.mdx:456]: Is it auto format? What tool are you using?

**Review by alex-tpom6oh — APPROVED** [review]
We need to standartize mdx formatting

File: pages/workspace/tracking-plan/publishing.mdx (hunk 1 above)
Comment (logason) [reply pages/workspace/tracking-plan/publishing.mdx:456]: Hmmm not sure if I'm using anything specific 🤔

**Pull request merged**